apps/wallet-dashboard/components/Dialogs/Assets/AssetDialog.tsx

```diff
@@ -66,16 +77,30 @@ export function AssetDialog({ onClose, asset }: AssetsDialogProps): JSX.Element
     }
     return (
         <Dialog open onOpenChange={onOpenChange}>
-            <FormikProvider value={formik}>
-                <>
-                    {view === AssetsDialogView.Details && (
-                        <DetailsView asset={asset} onClose={onOpenChange} onSend={onDetailsSend} />
-                    )}
-                    {view === AssetsDialogView.Send && (
-                        <SendView asset={asset} onClose={onOpenChange} onBack={onSendViewBack} />
-                    )}
-                </>
-            </FormikProvider>
+            <DialogLayout>
+                <FormikProvider value={formik}>
+                    <>
+                        {view === AssetsDialogView.Details && (
+                            <DetailsView
+                                asset={asset}
+                                onClose={onOpenChange}
+                                onSend={onDetailsSend}
+                            />
+                        )}
+                        {view === AssetsDialogView.Send && (
+                            <SendView
+                                asset={asset}
+                                onClose={onOpenChange}
+                                onBack={onSendViewBack}
+                            />
+                        )}
+
+                        {view === AssetsDialogView.TransactionDetails && !!digest ? (
+                            <TransactionDetailsView digest={digest} onClose={onOpenChange} />
+                        ) : null}
+                    </>
+                </FormikProvider>
+            </DialogLayout>
```

Comment:
we don't need views `DetailsView` and `TransactionDetailsView` inside the Formik Provider
```suggestion
           <DialogLayout>
                {view === AssetsDialogView.Details && (
                    <DetailsView asset={asset} onClose={onOpenChange} onSend={onDetailsSend} />
                )}
                {view === AssetsDialogView.Send && (
                    <FormikProvider value={formik}>
                        <SendView asset={asset} onClose={onOpenChange} onBack={onSendViewBack} />
                    </FormikProvider>
                )}
                {view === AssetsDialogView.TransactionDetails && !!digest ? (
                    <TransactionDetailsView digest={digest} onClose={onOpenChange} />
                ) : null}
            </DialogLayout>
```